app/src/docker/docker-compose.yml

```diff
@@ -101,6 +101,7 @@ services:
       RABBITMQ_PLACEHOLDERS_HOST: vro-api-rabbit-mq-service
       RABBITMQ_PLACEHOLDERS_USERNAME: ${RABBITMQ_PLACEHOLDERS_USERNAME}
       RABBITMQ_PLACEHOLDERS_USERPASSWORD: ${RABBITMQ_PLACEHOLDERS_USERPASSWORD}
+      VRO_NON_PROD: "$VRO_NON_PROD"
```

Comment:
Is it an environment variable ${VRO_NON_PROD}?  I don't see VRON_NON_PROD in app container's env list.